tests/main.nf.test

```diff
@@ -47,12 +47,30 @@ nextflow_pipeline {
             // check IRIDA Next JSON output
             def iridanext_json = path("$launchDir/results/iridanext.output.json").json
             def iridanext_samples = iridanext_json.files.samples
+            def iridanext_metadata = iridanext_json.metadata.samples
 
+            // output files
             assert iridanext_samples.SAMPLE2.findAll { it.path == "adjust/SAMPLE2_S_bracken_abundances.csv" }.size() == 1
             assert iridanext_samples.SAMPLE2.findAll { it.path == "krona/SAMPLE2.html" }.size() == 1
             assert iridanext_samples.SAMPLE2.findAll { it.path == "fastp/SAMPLE2.html" }.size() == 1
             assert iridanext_samples.SAMPLE2.findAll { it.path == "fastp/SAMPLE2_R1_trimmed.fastq.gz" }.size() == 1
             assert iridanext_samples.SAMPLE2.findAll { it.path == "fastp/SAMPLE2_R2_trimmed.fastq.gz" }.size() == 1
+
+            // output metadata
+            assert iridanext_metadata.SAMPLE2."abundance_1_name" == "Escherichia coli"
+            assert iridanext_metadata.SAMPLE2."abundance_1_ncbi_taxonomy_id" == "562"
+            assert iridanext_metadata.SAMPLE2."abundance_1_num_assigned_reads" == "631"
+            assert iridanext_metadata.SAMPLE2."abundance_1_fraction_total_reads" == "2.52531"
+            assert iridanext_metadata.SAMPLE2."abundance_2_name" == "Bacteroides fragilis"
+            assert iridanext_metadata.SAMPLE2."abundance_2_ncbi_taxonomy_id" == "817"
+            assert iridanext_metadata.SAMPLE2."abundance_2_num_assigned_reads" == "22"
+            assert iridanext_metadata.SAMPLE2."abundance_2_fraction_total_reads" == "0.08805"
+            assert iridanext_metadata.SAMPLE2."abundance_3_name" == "None"
```

Comment:
Could you add `assert` statements for `abundance_4` and `abundance_5` results (which should be all missing values)?

Reply:
Of course! Added here: [c739693](https://github.com/phac-nml/speciesabundance/pull/12/commits/c739693be421866c2c8a9b6f9a586caa91078eed)